Alignment/APEEstimation/test/SkimProducer/skimProducer_cfg.py

```diff
@@ -161,9 +153,6 @@
     process.MuSkim.src = 'TrackList'
     process.TriggerSelectionSequence *= process.TrackList
 
-import Alignment.CommonAlignment.tools.trackselectionRefitting as trackselRefit
```

Comment:
why do you remove the track selection?
shouldn't you just adapt it to the track collection you are using?